specification/addons/resource-manager/readme.md

```diff
@@ -26,9 +26,17 @@ These are the global settings for the Addons API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2017-05
+tag: package-2018-03
 ```
 
+### Tag: package-2018-03
+
+These settings apply only when `--tag=package-2018-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-03'
+input-file:
+- Microsoft.Addons/preview/2018-03-01/Addons.json
```

Comment:
that's not what your Swagger is called (`2018-03-01/addons-swagger.json`), either rename the Swagger or use correct name here

Reply:
My bad. Will update it in the next commit. Thanks!